plugins/plugin-chart-echarts/src/MixedTimeseries/controlPanel.tsx

```diff
@@ -70,7 +70,18 @@ function createQuerySection(label: string, controlSuffix: string): ControlPanelS
       [
         {
           name: `adhoc_filters${controlSuffix}`,
-          config: sharedControls.adhoc_filters,
+          config: {
+            ...sharedControls.adhoc_filters,
+            mapStateToProps: (state, controlState) => ({
+              ...(sharedControls?.adhoc_filters?.mapStateToProps?.(state, controlState) || {}),
```

Comment:
I think that null check of `sharedControls` is not necessary here, as we don't check the null-ness 2 lines above 🙂

Reply:
Got it 👍  I'm always unsure about these, hence I always end up padding everything with `?`